src/test/java/org/broadinstitute/hellbender/tools/walkers/annotator/VariantAnnotatorEngineUnitTest.java

```diff
@@ -90,127 +84,19 @@ public void testFinalizeAnnotations() throws Exception {
-        final List<InfoFieldAnnotation> knowInfoAnnos = ClassUtils.makeInstancesOfSubclasses(InfoFieldAnnotation.class, Annotation.class.getPackage());
-        Assert.assertEquals(vae.getGenotypeAnnotations().size(), knowGenoAnnos.size());
-        Assert.assertEquals(vae.getInfoAnnotations().size(), knowInfoAnnos.size());
-
-        final Set<VCFHeaderLine> vcfAnnotationDescriptions = vae.getVCFAnnotationDescriptions(false);
-        Assert.assertFalse(vcfAnnotationDescriptions.isEmpty());
-
-        Assert.assertFalse(vcfAnnotationDescriptions.contains(VCFStandardHeaderLines.getInfoLine(VCFConstants.DBSNP_KEY)));
-        Assert.assertTrue(vcfAnnotationDescriptions.contains(VCFStandardHeaderLines.getInfoLine(VCFConstants.DEPTH_KEY)));          //yes DP
-        Assert.assertTrue(vcfAnnotationDescriptions.contains(VCFStandardHeaderLines.getInfoLine(VCFConstants.ALLELE_COUNT_KEY)));   //yes AC
-    }
-
-    @Test
-    public void testAllMinusCoverage(){
-        final List<String> annotationsToExclude= Arrays.asList(Coverage.class.getSimpleName());
-        final FeatureInput<VariantContext> dbSNPBinding = null;
-        final List<FeatureInput<VariantContext>> features = Collections.emptyList();
-        final VariantAnnotatorEngine vae = VariantAnnotatorEngine.ofAllMinusExcluded(annotationsToExclude, dbSNPBinding, features, false);
-        Assert.assertFalse(vae.getVCFAnnotationDescriptions(false).contains(null));
-        Assert.assertFalse(vae.getGenotypeAnnotations().isEmpty());
-        Assert.assertFalse(vae.getInfoAnnotations().isEmpty());
-
-        final Set<VCFHeaderLine> vcfAnnotationDescriptions = vae.getVCFAnnotationDescriptions(false);
-        Assert.assertFalse(vcfAnnotationDescriptions.isEmpty());
-
-        Assert.assertFalse(vcfAnnotationDescriptions.contains(VCFStandardHeaderLines.getInfoLine(VCFConstants.DBSNP_KEY)));
-        Assert.assertFalse(vcfAnnotationDescriptions.contains(VCFStandardHeaderLines.getInfoLine(VCFConstants.DEPTH_KEY))); //no DP
-        Assert.assertTrue(vcfAnnotationDescriptions.contains(VCFStandardHeaderLines.getInfoLine(VCFConstants.ALLELE_COUNT_KEY)));  //yes AC
-    }
-
-    @Test(expectedExceptions = CommandLineException.BadArgumentValue.class)
-    public void testBadAnnot(){
-        final List<String> annotationsToExclude= Collections.emptyList();
-        final FeatureInput<VariantContext> dbSNPBinding = null;
-        final List<String> annotationGroupsToUse = Collections.emptyList();
-        final List<String> annotationsToUse = Arrays.asList("fred");
-        final List<FeatureInput<VariantContext>> features = Collections.emptyList();
-        final VariantAnnotatorEngine vae = VariantAnnotatorEngine.ofSelectedMinusExcluded(annotationGroupsToUse, annotationsToUse, annotationsToExclude, dbSNPBinding, features);
-        Assert.assertFalse(vae.getGenotypeAnnotations().isEmpty());
-        Assert.assertFalse(vae.getInfoAnnotations().isEmpty());
-    }
-
```

Comment:
It seems like we dropped a lot of tests here - are there corresponding tests elsewhere that are redundant with these ?

Reply:
yes, they were mostly tests i cannibalized and moved to AnnotationsBarclayPlugin unit tests. The tests that moved are almost exclusively about ofXminusY behavior which the plugin handles now.